node/resolvers/items.ts

```diff
@@ -174,10 +201,10 @@ export const mutations = {
 
     const cleanItems = orderItems.map(({ id, ...rest }) => rest)
 
-    if (cleanItems.some((item: OrderFormItemInput) => !item.index)) {
-      const orderForm = await checkout.orderForm(orderFormId!)
+    const currentOrderForm = await checkout.orderForm(orderFormId!)
```

Comment:
Why this was moved?

Reply:
I moved it up in scope because of that bit of code that handled subscriptionData updates. Now this change is unnecessary, nice catch! :D